app/models/json_builders/identity_json_builder.rb

```diff
@@ -95,7 +87,7 @@ def build_site_identity_json
 
   def build_twitch_identity_json
     Jbuilder.encode do |json|
-      json.publisher      @publisher_name
+      json.publisher      Channel::TWITCH
```

Comment:
I don't think this is right? shouldn't it be `@publisher_name` like you had before